channels/chan_usbradio.c

```diff
@@ -4388,6 +4388,37 @@ static void tune_rxctcss(int fd, struct chan_usbradio_pvt *o, int intflag)
 	o->pmrChan->b.tuning = 0;
 }
 
+/*!
+ * \brief Update the tune settings to the configuration file.
+ * \param filename	The configuration file being updated (e.g. "usbradio.conf").
+ * \param category	The category being updated (e.g. "12345").
+ * \param variable	The variable being updated (e.g. "rxboost").
+ * \param value		The value being updated (e.g. "yes").
+ * \retval 0		If successful.
+ * \retval -1		If unsuccessful.
+ */
+static int tune_variable_update(const char *filename, struct ast_category *category,
+								const char *variable, const char *value)
+{
+	int	res;
+	struct ast_variable *var;
+
+	res = ast_variable_update(category, variable, value, NULL, 0);
+	if (res == 0) {
+		return 0;
+	}
+
+		/* if we could not find/update the variable, create new */
+		var = ast_variable_new(variable, value, filename);
+	if (var == NULL) {
+		return -1;
+	}
+
+			/* and append */
```

Comment:
Same thing with indentation - just need to remove some tabs

Reply:
tabstops=4 vs. tabstops=8

Reply:
A tab is a tab, the alignment should be correct no matter what the tabstop is.
I took another look and I think there is an extra tab on some lines when there shouldn't be. e.g. 3220, 3229, etc.

Reply:
Fixed.